classes/item.rb

```diff
@@ -2,12 +2,17 @@
 
 class Item
   attr_reader :id, :archived
-  attr_accessor :publish_date
+  attr_accessor :publish_date, :author
 
-  def initialize(id: Rand.random(1.1000), publish_date: Date.today.year, archived: false)
-    @id = id
+  def initialize(publish_date, archived, id: Random.rand)
     @publish_date = publish_date
     @archived = archived
+    @id = id
```

Comment:
[Optional suggestion] : 
Adding `@author` here would improve readbility, since all instance variables are readily available within the `initialize` method.
```suggestion
    @id = id
    @author = nil
```